etc/events.xml

```diff
@@ -9,6 +9,7 @@
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Event/etc/events.xsd">
     <event name="sales_order_save_after">
         <observer name="metrilo_event_order" instance="Metrilo\Analytics\Observer\Order"/>
+        <observer name="metrilo_identify_customer_order" instance="Metrilo\Analytics\Observer\IdentifyUser"/>
```

Comment:
why do you call the person once `customer` and once `user`?

Reply:
Hmm didn't notice it until now, but it's more appropriate to call that person user. Fixing it.

Reply:
In Metrilo we use the term `customer` and as I can see, in Magento they use the term `customer`. Which one do you think is better, "user" or "customer"?

Reply:
Every user that has provided email at some point of the browsing is actually a customer (currently checking out, or logged in user with potential order history otherwise why bother creating account). So following that logic it makes more sense to call it customer, since both user and customer are referencing same entity. To be honest I'm not really sure why i picked the  'user' naming approach instead of 'customer'.

Reply:
👍